shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/config/schema/impl/DataSourceGeneratedSchemaConfiguration.java

```diff
@@ -41,14 +41,27 @@
     private final Map<String, DataSource> dataSources;
     
     private final Collection<RuleConfiguration> ruleConfigurations;
+
+    private final Map<String, DataSourceProperties> dataSourceProperties;
     
     public DataSourceGeneratedSchemaConfiguration(final Map<String, DataSourceConfiguration> dataSources, final Collection<RuleConfiguration> ruleConfigurations) {
         this.dataSources = DataSourcePoolCreator.create(createDataSourcePropertiesMap(dataSources));
         this.ruleConfigurations = ruleConfigurations;
+        this.dataSourceProperties = createDataSourcePropertiesMap(dataSources);
```

Comment:
`this.dataSourceProperties` can be change to `dataSourceProperties`, `this.` is unnecessary here

Reply:
Fix in https://github.com/apache/shardingsphere/pull/16632